### ISSUE TITLE
Machine Learning Projects

### ISSUE THREAD

**Issue by benlapETS** [labesoft/ml-projects#1]
In this project, we complete [machine learning sub-projects](https://data-flair.training/blogs/machine-learning-project-ideas/) for beginners, intermediates, and experts to gain real-world experience of this growing technology in 2021. These machine learning project ideas helps us in learning all the practicalities that we need to succeed in the industry.

These machine learning projects are developed in Python:

- [x] #2 Create an Image Cartoonifyer with OpenCV
- [x] #3 Create a Handwritten Digit Recognition using Python
- [x] #4 Create a Stock Price Prediction Application
- [x] #5 Create an Detecting Fake News Application
- [x] #6 Create a Colour Detection using Pandas & OpenCV
- [x] #7 Create a Real-Time Face Mask Detector with Python, OpenCV, Keras
- [x] #8 Create a Music Genre Classification system
- [x] #9 Create a Sign Language Recognition Using OpenCV
- [x] #10 Create a Speech Emotion Recognition with librosa
- [x] #11 Create a Handwritten Character Recognition with Neural Network
- [x] #12 Create a Breast Cancer Classification with Deep Learning
- [x] #13 Create a Gender and Age Detection with OpenCV
- [x] #14 Create an Image Segmentation with Machine Learning
- [x] #15 Create a Driver Drowsiness Detection System with OpenCV & Keras
- [x] #16 Create a Real-time Human Detection & Counting

**Comment by NourZuraiki** [comment]
What is the issue?

**Comment by KALYANSAI-3114** [comment]
I would like to work on this issue. Can you assign it to me?

**Comment by benlapETS** [comment]
don’t bother working on these, only old demo issue … we could keep in touch though as real valuable project will emerge in 2026 with our new initiatives
